truss/templates/server/model_wrapper.py

```diff
@@ -515,17 +525,17 @@ def _initialize_environment_before_load(self):
             self._model.setup_environment(environment_json)
             self._environment = environment_json
 
-    async def setup_environment(self, environment: Optional[dict]):
+    async def setup_environment(self, environment: Optional[dict]) -> None:
```

Comment:
Small cleanup, nothing ever looks at the return value of `setup_environment` so I explicitly made the signature `None`